microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -3,89 +3,119 @@ description: Version Models used for Microsoft Edge WebView2
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 05/06/2021
+ms.date: 07/13/2021
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: webview
 keywords: IWebView2, IWebView2WebView, webview2, webview, wpf apps, wpf, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html
 ---
-# Understand WebView2 SDK versions  
+# Understand WebView2 SDK versions
 
 New versions of the WebView2 SDK are shipped at the same general cadence as the Microsoft Edge \(Chromium\) browser, which is approximately every six weeks.  
 
-## Release and prerelease package  
+When you develop and distribute a WebView2 app, the client must at least have a version of the Runtime that can use the core APIs and create a WebView2.  The client ideally should have a version of the Runtime that supports all of the latest APIs that are in the SDK version you use to develop the app.  You must coordinate the WebView2 SDK version that you use for development, and the WebView2 Runtime version that's installed on client machines.
 
-The WebView2 NuGet package contains both a release and pre-release package.  
+*  Core API functionality requires that the client has WebView2 Runtime version `86.0.616.0` or newer.
+*  Full support for the latest APIs in an SDK release requires that the Runtime on the client have a version number greater than or equal to the SDK version.
 
-The **release package** is forward compatible and contains the following components.  
 
-*   [Win32 C/C++ APIs][ReferenceWin32]
-*   .NET APIs:  [WPF][DotnetMicrosoftWebWebview2WpfNamespace], [WinForms][DotnetMicrosoftWebWebview2WinformsNamespace], and [Core][DotnetMicrosoftWebWebview2CoreNamespace].  
-    
-APIs in the SDK are fully supported.  
+## Release and prerelease packages
+
+The NuGet package for the WebView2 SDK contains both a release and prerelease package.
 
-The **prerelease package** is a superset of the release package with [Experimental APIs](#experimental-apis).  Avoid using the prerelease SDK to build production apps.  
+*  The _release_ package of the SDK contains all of the stable Win32 C/C++ and .NET APIs.  APIs that are in the release package are fully supported.  The release package is forward-compatible and contains the following components:
+    *  [Win32 C/C++ APIs][ReferenceWin32].
+    *  .NET APIs:  [WPF][DotnetMicrosoftWebWebview2WpfNamespace], [WinForms][DotnetMicrosoftWebWebview2WinformsNamespace], and [Core][DotnetMicrosoftWebWebview2CoreNamespace].  
 
-### Roadmap  
+*  The _prerelease_ package of the SDK is a superset of the release package, with additional, [Experimental APIs](#experimental-apis) that are subject to change based on your feedback.  Avoid using the prerelease version of the WebView2 SDK to build production apps.
 
-The release package contains all of the stable, supported Win32 C/C++ and .NET APIs.  The prerelease package contains experimental APIs that are subject to change based on your feedback.  
 
 ## Experimental APIs  
```

Comment:
I would move the Experimental API stuff you had in the distribution doc (see `### Experimental APIs are not guaranteed to be forward-compatible`) to here.